e2e-tests/gef/cypress/fixtures/mocks/mock-user-maker.js

```diff
@@ -1,8 +1,8 @@
 exports.MOCK_USER_MAKER = {
   username: 'BANK1_MAKER1',
   password: 'AbC!2345',
-  firstname: 'Tamil',
-  surname: 'Rahani',
+  firstname: 'First',
```

Comment:
it may not be part of this PR, but should we start using the `chance` package for these values? i.e. `chance.first()`

Reply:
Good shout I'll raise a new PR once this is merged